boxtree/cost.py

```diff
@@ -378,17 +387,29 @@ def aggregate_over_boxes(self, per_box_result):
         """
         pass
 
-    def fmm_cost_factors_for_kernels_from_model(self, nlevels, xlat_cost, context):
+    @staticmethod
+    def cost_factors_to_dev(cost_factors, queue):
```

Comment:
Would it be practical to have this return a new object rather than modify an existing one?